### DISCUSSION TITLE
How to add a logo or image to the footer?

### DISCUSSION THREAD

**Discussion by toorajpro** [arthelokyo/astrowind#400] [Q&A]
The footer links are defined in the `navigation.js`. But everything there is text and links.
How can I add for example the logo of my website there?

**Comment by Harold-Anderson** [comment]
To add the logo to the footer, you can just change the first few lines of the File `Footer.astro`  to:
```
import { Icon } from 'astro-icon/components';
import { SITE } from 'astrowind:config';
import Logo from '../Logo.astro';
import { getHomePermalink } from '~/utils/permalinks';
```

Then your logo will be imported.  Make sure to edit the file `Logo.astro` as follows:

```
// import { SITE } from 'astrowind:config';
import { Image } from 'astro:assets';
import ACFOLogo from '../assets/images/Logo_Black.png'

---

<span
  class="self-center ml-2 rtl:ml-0 rtl:mr-2 text-2xl md:text-xl font-bold text-gray-900 whitespace-nowrap dark:text-white"
>
<Image 
   src={ACFOLogo}
   width = {ACFOLogo.width / 8}
   alt="Anderson CFO logo"
   />

</span>
```
 
Obviously, you have to use your actual logo instead of my Logo_Black.png file

However, I do second your request to be able to add other images, such as logos of partners.
